src/frontend/query_commands.ml

```diff
@@ -351,6 +351,76 @@ let dispatch pipeline (type a) : a Query_protocol.t -> a = function
         | `Found { file; location; _ } -> `Found (Some file, location.loc_start)
         | `File_not_found { file = reason; _ } -> `File_not_found reason)
     end
+  | Locate_types pos -> (
+    let typer = Mpipeline.typer_result pipeline in
+    let verbosity = verbosity pipeline in
+    let local_defs = Mtyper.get_typedtree typer in
+    let structures = Mbrowse.of_typedtree local_defs in
+    let pos = Mpipeline.get_lexing_pos pipeline pos in
+    let result =
+      let ( let* ) opt f = Option.bind opt ~f in
+      let ( let+ ) opt f = Option.map opt ~f in
```

Comment:
Could you move `let*` and `let+` (and maybe `and+`) into `util/std.ml` inside `Option.Infix` and just locally open `Option.Infix` here?

Reply:
Ah yes good call